sdk/storage/azure-storage-file-share/tests/test_share.py

```diff
@@ -1887,6 +1890,114 @@ def test_smb_directory_lease(self, **kwargs):
         assert shares[0].enable_smb_directory_lease == False
         assert shares[1].enable_smb_directory_lease == True
 
+    @pytest.mark.live_test_only
```

Comment:
Any reason these need to be Live only? If you keep them Live only, you don't need `get_datetime_variable` and the `variables` as that's for recordings only.